client/my-sites/hosting/php-version-card/index.js

```diff
@@ -163,13 +166,15 @@ const PhpVersionCard = ( {
 			return;
 		}
 
-		const isButtonDisabled = ! selectedPhpVersion || selectedPhpVersion === currentPhpVersion;
+		const isButtonDisabled =
```

Comment:
It'd be nice to also avoid the PHP version request when the site is not Atomic (`disabled === true`).